frame/contracts/src/tests.rs

```diff
@@ -2766,8 +2766,7 @@ fn gas_estimation_nested_call_fixed_limit() {
 }
 
 #[test]
-fn gas_estimation_call_runtime() {
-	use codec::Decode;
```

Comment:
I think the test is stuff useful. It re-enters pallet-contracts this is why it broke. But this can easily be fixed: We just need to find another dispatchable with a big difference in pre dispatch weight and post dispatch weight. Those should exist.

Reply:
done